### ISSUE TITLE
Staff counter in staff window is not properly translated into German

### ISSUE THREAD

**Issue by Wuzzy2** [OpenRCT2/Localisation#2593]
In the Staff window, at the bottom there is the number of staff shown. In English it says something like “15 handymen”.

However, in German, it only shows the number, e.g. “15”.

I believe this is a mistake in the translation file (rather than OpenRCT2) but unfortunately, I don't know which is the offending string that needs to be corrected. I have not found it yet. Can someone help me out here, please?

**Comment by tupaschoal** [comment]
Indeed, and the issue only happens since v0.4.3, with all languages other than English. I'll move this issue to `OpenRCT2` repo instead.
